armi/runLog.py

```diff
@@ -184,16 +183,13 @@ def log(self, msgType, msg, single=False, label=None, **kwargs):
         self.logger.log(msgLevel, msg, single=single, label=label)
 
     def getDuplicatesFilter(self):
-        """
-        The top-level ARMI logger should have a no duplicates filter
-        If it exists, find it.
-        """
+        """If it exists, find the top-level ARMI logger 'should have a no duplicates' filter."""
         if not self.logger or not isinstance(self.logger, logging.Logger):
             return None
 
         return self.logger.getDuplicatesFilter()
 
-    def clearSingleWarnings(self):
+    def clearSingleLogs(self):
         """Reset the single warned list so we get messages again."""
```

Comment:
update the docstring to be about logs and not just warnings

Reply:
done